packages/react-scripts/template/src/registerServiceWorker.js

```diff
@@ -7,29 +7,30 @@
 
 // To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
 // This link also includes instructions on opting out of this behavior.
+const {URL, fetch, location, addEventListener} = window;
```

Comment:
I would prefer we just explicitly keep referring to them as `window.<whatever>` rather than destructure here.